modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -1116,6 +1250,68 @@ private void _assertNotificationQueueEntryTermValues(
 			notificationQueueEntry);
 	}
 
+	private Map<String, Object> _createFreeMarkerTermValuesMap(
+			ObjectDefinition objectDefinition, PersistedModel persistedModel,
+			Set<String> selectedTermNames)
```

Comment:
@devpedrotavares here we can rename to `termNames` because we don't have a conflict here.